File: svc-event/src/test/java/com/beancounter/event/EventBootTests.java
package com.beancounter.event;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;
import org.springframework.web.context.WebApplicationContext;

@SpringBootTest
@Tag("slow")
@ActiveProfiles("test")
class EventBootTests {
  private final WebApplicationContext context;

  @Autowired
  private EventBootTests(WebApplicationContext webApplicationContext) {
    super();
    this.context = webApplicationContext;
  }

  @Test
  void contextLoads() {
    assertThat(context).isNotNull();
  }

}

File: svc-event/src/test/kotlin/com/beancounter/event/EventBootTests.kt
package com.beancounter.event

import com.beancounter.auth.AutoConfigureMockAuth
import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Tag
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.security.oauth2.jwt.JwtDecoder
import org.springframework.test.context.ActiveProfiles
import org.springframework.test.context.bean.override.mockito.MockitoBean

/**
 * Test Context loads.
 */
@SpringBootTest
@Tag("slow")
@ActiveProfiles("test")
@AutoConfigureMockAuth
internal class EventBootTests {
    @MockitoBean
    private lateinit var jwtDecoder: JwtDecoder

    @Test
    fun contextLoads() {
        Assertions.assertThat(jwtDecoder).isNotNull
    }
}